packages/animations/example/lib/fade_scale_transition.dart

```diff
@@ -85,6 +85,7 @@ class _FadeScaleTransitionDemoState extends State<FadeScaleTransitionDemo>
                   onPressed: () {
                     showModal<void>(
                       context: context,
+                      configuration: const FadeScaleTransitionConfiguration(),
```

Comment:
Is this necessary? I think `showModal` [currently defaults to FadeScaleTransitionConfiguration](https://github.com/flutter/packages/blob/master/packages/animations/lib/src/modal.dart#L51).

Reply:
I did copy the code from the example then past it into my project to see how it work. But without the configuration object. My code could not run and threw me an error. 
I'm using flutter v1.17.1 and after jumping into `showModal` function, I could not find out the configuration object so that I decided to send a PR to help
Code in `showModal` function
```
Future<T> showModal<T>({
  @required BuildContext context,
  @required ModalConfiguration configuration,
  bool useRootNavigator = true,
  WidgetBuilder builder,
})
```

Reply:
Did you test this with the master branch for `flutter/packages/`? The update has not been pushed onto pub yet AFAIK

Reply:
Also, this code change only updates the example app and does not make any changes to the code in the source code for `showModal`